src/mavsdk/plugins/mission_raw_server/mission_raw_server_impl.cpp

```diff
@@ -279,9 +320,21 @@ void MissionRawServerImpl::unsubscribe_incoming_mission(
     _incoming_mission_callbacks.unsubscribe(handle);
 }
 
+MissionRawServer::OutgoingMissionHandle MissionRawServerImpl::subscribe_outgoing_mission(
+    const MissionRawServer::OutgoingMissionCallback& callback)
+{
+    return _outgoing_mission_callbacks.subscribe(callback);
+}
+
+void MissionRawServerImpl::unsubscribe_outgoing_mission(
+    MissionRawServer::OutgoingMissionHandle handle)
+{
+    _outgoing_mission_callbacks.unsubscribe(handle);
+}
+
 MissionRawServer::MissionPlan MissionRawServerImpl::incoming_mission() const
 {
-    // FIXME: this doesn't look right.
+    // TO-DO
```

Comment:
Nevermind, we should just make that one ASYNC only, I think.

Reply:
Ha yeah I actually did consider adding support for this as part of the PR but `current_item_changed()` and `clear_all()` both have the same issue, I figured all three should be done at the same time and I had already let this balloon enough. I agree it might make sense to just make all of them ASYNC though, if you want blocking behavior on the app side you can always set up promises for them. Would you like me to do that? It's easy enough to just update the proto and regen.

Reply:
Exactly. I can do it in a follow up PR if you don't beat me to it.

Reply:
No worries, I can roll it in here along with all the other changes discussed tomorrow morning US/Pacific time